core/org.wso2.carbon.server.admin.ui/src/main/resources/web/server-admin/index.jsp

```diff
@@ -34,12 +35,27 @@
     <h2><fmt:message key="shutdown.restart.server"/></h2>
     <div id="workArea">
         <div id="output" style="display:none;"></div>
-
+    
+        <%
+            ServerConfiguration serverConfig = ServerConfiguration.getInstance();
+            String disableShutdown = serverConfig.getFirstProperty("disableShutdownRestartFromUI.disableShutdown");
+            String disableRestart = serverConfig.getFirstProperty("disableShutdownRestartFromUI.disableRestart");
```

Comment:
disableShutdownRestartFromUI.disableShutdown 

This property does not make sense same information is there "disableShutdownRestart" , "disableShutdown"any Shall we change it

ex: shutdownRestartFromUI.enable=false

Reply:
The relevant configuration would be as follows.
``<disableShutdownRestartFromUI>
          <disableShutdown>true</disableShutdown>
          <disableRestart>true</disableRestart>
      </disableShutdownRestartFromUI>``